ethers-core/src/types/chain.rs

```diff
@@ -140,6 +140,8 @@ pub enum Chain {
     CantoTestnet = 740,
 
     Boba = 288,
+
+    ZKSync = 324,
```

Comment:
this needs to be renamed to lowercase for serde and strum, see the comments at the top of the file on how to do it